### ISSUE TITLE
GH actions: no `rmarkdown`?

### ISSUE THREAD

**Issue by trevorcampbell** [UBC-STAT/stat-406#21]
Every few runs of the github actions workflow results in an error where the runner complains that `rmarkdown` and `knitr` aren't installed. 

https://github.com/UBC-STAT/stat-406/actions/runs/10860357801/job/30140845043

It doesn't happen every time, so it's probably related to whether built files in `_freeze` etc are being pushed as well.

**Comment by trevorcampbell** [comment]
Yep, my guess is probably right: see https://github.com/quarto-dev/quarto-cli/issues/5870

**Comment by trevorcampbell** [comment]
On the one hand, it may be worth fixing this. OTOH the Stat406 package is so huge that we probably don't want a runner to install it every time it triggers. So probably the best thing to do is either to make a docker image that we can run the workflow on and put it on DockerHub, or to just leave it alone (but document that you need to fully commit all built files to get a working site rebuild)

**Comment by dajmcdon** [comment]
* I always built locally, more because I didn't want the runner to get bogged down rerunning some simulation.
* If you have it build on the runner, I think the runner should just install from cache, so the package size won't be a problem. Not positive though.
